src/bindgen/parser.rs

```diff
@@ -318,33 +318,42 @@ impl<'a> Parser<'a> {
                     } else if next_mod_path2.exists() {
                         self.parse_mod(pkg, next_mod_path2.as_path())?;
                     } else {
-                        // Last chance to find a module path
-                        let mut path_attr_found = false;
-                        for attr in &item.attrs {
-                            if let Some(syn::Meta::NameValue(syn::MetaNameValue {
-                                ident,
-                                lit,
-                                ..
-                            })) = attr.interpret_meta()
-                            {
-                                match lit {
-                                    syn::Lit::Str(ref path) if ident == "path" => {
-                                        path_attr_found = true;
-                                        self.parse_mod(pkg, &mod_dir.join(path.value()))?;
-                                        break;
-                                    }
+                        let next_mod_path1 = mod_dir.join(next_mod_name.clone() + ".rs");
+                        let next_mod_path2 = mod_dir.join(next_mod_name.clone()).join("mod.rs");
```

Comment:
This looks like a rebase mistake? The logic is the same as a couple lines before.

Reply:
:man_facepalming: